### DISCUSSION TITLE
🧩 FEATURE REQUEST: Ability to follow other users

### DISCUSSION THREAD

**Discussion by bymicah** [dailydotdev/daily#613] [Feature Requests]
### Description

I would like to have the ability to follow other users in which if they posted a new blog/article, I will be able to see it on my feed. I should also see the complete list of the people I followed and those who followed me, and the number of my following and followers.

### Solution

**Example**

This is what I see, in the perspective of other users, as of now:

<img src="https://user-images.githubusercontent.com/84730903/155646930-a2039c5e-7e07-4d1d-9365-bfbb00fadcb3.png">

Expected output is something like this one:

<img src="https://user-images.githubusercontent.com/84730903/155647428-60fd1cf9-9f41-4490-8780-394126c560e7.png">

If I clicked the follow button, I should be able to follow that user. If I clicked the 5 followers, I should be able to see the list of those followers. If I clicked the 5 following, I should be able to see the list of following.

Thank you very much!

**Comment by idoshamun** [comment]
Hi @purplemicah,

Thanks a lot for the suggestion and the mock-ups.
We are in discussion about such a feature but it's not in on near-future plans.
Anyway, we will update once we have some news 😁

**Comment by bymicah** [comment]
> Hi @purplemicah,
> 
> Thanks a lot for the suggestion and the mock-ups. We are in discussion about such a feature but it's not in on near-future plans. Anyway, we will update once we have some news 😁

Awesome! Thanks. I'll wait for that.
